app/services/ipam/base.rb

```diff
@@ -17,6 +17,18 @@ def initialize(opts = {})
       normalize_mac!
     end
 
+    def subnet_range
+      @subnet_range ||= begin
+        subnet_range = IPAddr.new("#{subnet.network}/#{subnet.mask}", subnet.family).to_range
+        # exclude first element - network
+        from = subnet.from.present? ? IPAddr.new(subnet.from) : subnet_range.first(2).last
+        # exclude last element - broadcast
```

Comment:
I agree it's not in the scope of this PR and that IPv6 generally won't miss those IPs because the subnets are WAY bigger.